packages/matchbox/src/components/Tag/styles.js

```diff
@@ -0,0 +1,150 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export const tagBase = props => `
+  display: inline-flex;
+  align-items: center;
+  white-space: nowrap;
+  padding: 1px ${tokens.spacing_300};
+  padding-right: ${props.hasRemove ? '1px' : tokens.spacing_300};
+  border-radius: ${tokens.borderRadius_pill};
+  line-height: ${tokens.lineHeight_200};
+  font-size: ${tokens.fontSize_200};
+  font-weight: ${tokens.fontWeight_normal};
+`;
+
+// Includes button reset styles
+// Key events (space, enter) inherit the onClick handler through <button>
+export const closeBase = () => `
+  border: none;
+  background: none;
+  padding: 0;
+  display: inline-flex;
+  width: 1.25rem;
+  height: 1.25rem;
+  align-items: center;
+  justify-content: center;
+  margin-left: ${tokens.spacing_100};
+  border-radius: ${tokens.borderRadius_circle};
+  transition-duration: 0.15s;
+  transition-property: background;
+`;
+
+// This margin is a little funky, however, appears to be the best route to take to add
+// spacing between icons and other tag content. Ideally, if Tag instances do not have bare
+// strings inside, then the sibling combinator could be used (i.e., `> * + svg { margin-left: 2px; }`),
+// however, that's not possible as bare strings do not count as sibling elements.
+export const content = () => `
+  & > svg {
+    margin: 0 2px;
+  }
+`;
+
+export const tagColor = props => {
+  let bg = '';
+  let border = '';
+  let color = '';
+
+  switch (props.tagColor) {
+    case 'orange':
+    case 'blue':
+    case 'navy':
+      bg = tokens.color_blue_300;
+      border = tokens.color_blue_500;
+      color = tokens.color_blue_800;
+      break;
+    case 'green':
+      bg = tokens.color_green_300;
+      border = tokens.color_green_500;
+      color = tokens.color_green_800;
+      break;
```

Comment:
Looks like this one isn't meeting a11y muster. Only one!

<img width="171" alt="Screen Shot 2020-02-04 at 4 11 55 PM" src="https://user-images.githubusercontent.com/3613392/73787370-266c6e80-4769-11ea-8657-0a095024afbb.png">